src/lib.rs

```diff
@@ -11,29 +12,51 @@ use tracing_subscriber::registry::LookupSpan;
 
 mod xray_daemon;
 
+/// Add `aws.xray.trace_id` as a field to a tracing span to designate it as an
+/// X-Ray segment. Set the value of this field to either a
+/// [fresh trace id][trace_id::new], or one
+/// [parsed from HTTP headers][trace_id::from_headers].
+pub const TRACE_ID_FIELD: &'static str = "aws.xray.trace_id";
```

Comment:
can you run clippy? i think it would warn about not needing static here because that is already implied by const